src/sentry/receivers/useremail.py

```diff
@@ -19,3 +21,20 @@ def create_user_email(instance, created, **kwargs):
     dispatch_uid="create_user_email",
     weak=False
 )
+
+
+@email_verified.connect(weak=False)
+def verify_newsletter_subscription(email, **kwargs):
+    from sentry.app import newsletter
+
+    if not newsletter.enabled:
+        return
+
+    if not email.is_primary():
+        return
+
+    newsletter.update_subscription(
+        email.user,
```

Comment:
This isn't the type you expect — [`email` is a string](https://github.com/getsentry/sentry/blob/master/src/sentry/web/frontend/accounts.py#L211), not a `UserEmail` object.